data-raw/phenotypes.R

```diff
@@ -56,7 +59,12 @@ phenotype_data <- phenotype_files %>%
 
 ## Biospecimen metadata ----
 
-# synGet("syn18876530", version = 6, downloadLocation = here::here("data-raw", "pathology"))
+biospecimen_id <- "syn18876530"
```

Comment:
I'm assuming these hard-coded locations don't change often?

Reply:
No I don't think so! I would be surprised if they changed